rpa-manager/modules/ROOT/pages/processoperations-deploymentmap.adoc

```diff
@@ -18,18 +29,58 @@ The interrelationships of processes, RPA Bots and Configurations are mapped by a
 |View the process deployment map
 |Process Deployment Map Open
 
+|Execute, pause or stop configurations
+|Process Automation Open
+
 |===
 
 * You can only view processes for which you are part of the process team.
-* To execute, pause or stop configurations obtain the needed team member permissions for the production phase.
+* To execute, pause, or stop configurations, obtain the needed team member permissions for the respective phase.
+* You cannot interact with invokable configurations.
+* The deployment map shows only bots that have configurations deployed.
+
+== Filter Configurations by Status
+
+To show a subset of the map, filter the configurations by status.
+
+You can use the following filters:
+
+* Deployed:
++
+Show all process-configuration-bot combinations in which the configuration is deployed.
+* Undeployed
++
+Show all process-configuration-bot combinations in which the configuration is not deployed.
+* Paused 
++
+Show all process-configuration-bot combinations in which the configuration is deployed.
+* Errors and warnings
++ 
+Show all process-configuration-bot combinations in which errors or warnings occur.
+
+You can refine the filter results using the same filters again.
+
+To filter configurations by status:
+
+. Open the *Deployment Map* view in the *Process Operations* module.
+. Click the *Filter by state* icon.
+. Select a filter.
+. To refine the result, deselect the filters that you do not want to use on the shown subset.
+
+The deployment map view is updated.
+
+To show the complete map again, click *Reset Filters*.
 
 == Manage Configuration Executions
 
-The actions *Deploy*, *Pause*, *Continue*, and *Revoke process* each refer to a single configuration-bot combination. The action does not change identical configurations that are deployed to a different bot or different configurations that are deployed to the same bot.
+The actions *Start*, *Deploy*, *Pause*, *Continue*, and *Revoke process* each refer to a single configuration-bot combination. The action does not change identical configurations that are deployed to a different bot or different configurations that are deployed to the same bot.
```

Comment:
The "each" is not needed:
The actions *Start*, *Deploy*, *Pause*, *Continue*, and *Revoke process* refer to a single configuration-bot combination.